website/docs/reference/snapshot-configs.md

```diff
@@ -301,3 +301,48 @@ snapshots:
 ```
 
 </File>
+
+<VersionBlock firstVersion="1.9">
+
+#### Define column names in snapshot table
+
+Use the following configuration to define the names of the columns created in the [snapshot table](/docs/build/snapshots#snapshot-meta-fields).
+
+<File name='dbt_project.yml'>
```

Comment:
let's also show the example from the issue on how to do this in the config block:

```
{{
    config(
      target_database='analytics',
      target_schema='snapshots',
      unique_key='id',

      strategy='timestamp',
      updated_at='updated_at',
      
      snapshot_meta_column_names={
        dbt_valid_from: start_date,
        dbt_valid_to: end_date,
        dbt_scd_id: scd_id,
        dbt_updated_at: updated_date
      }
    )
}}
```

Reply:
we have some tab thing for other examples so you can see how to write a config in different places